packages/react/scripts/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Thinking about it, we can just ignore the `scripts` from the typescript build since the script is run using `ts-node` (also trying to use a single tsconfig with multiple routes caused annoying issues with the exported types).